dist/images/ovnkube.sh

```diff
@@ -242,7 +231,6 @@ ovs_ready () {
   return 0
 }
 
-# version 1 daemonset compatibility
 # ovn must be up and initialized before ovn-controller, init-node, and init-master
 # This waits for northd to come up
 northd_ready () {
```

Comment:
northd_ready() function is not used anywhere, so you can remove it.

Reply:
OK